lib/ActivityStreams.js

```diff
@@ -97,6 +98,7 @@ function ActivityStreams(metadataStore, options = {}) {
   );
 
   this._previewProvider = new PreviewProvider(this._tabTracker, metadataStore);
+  this._pageScraper = new PageScraper();
```

Comment:
`PageScraper` should only be instantiated if the pref is on, otherwise it's consuming memory even if it's not being used. If we default to off, we should not instantiate